config/packages/doctrine.yaml

```diff
@@ -24,9 +24,9 @@ doctrine:
         naming_strategy: doctrine.orm.naming_strategy.underscore
         auto_mapping: true
         mappings:
-            Domain:
+            Projections:
                 is_bundle: false
                 type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
-                prefix: 'App\Domain'
-                alias: Domain
+                dir: '%kernel.project_dir%/config/mapping/orm/projections'
+                prefix: 'App'
```

Comment:
`doctrine:schema:validate` is failing due to mapping issues.

I think the best way to fix it is:
```
            Domain:
                is_bundle: false
                type: yml
                dir: '%kernel.project_dir%/config/mapping/orm/domain'
                prefix: 'App\Domain'
                alias: Domain
            Projections:
                is_bundle: false
                type: yml
                dir: '%kernel.project_dir%/config/mapping/orm/projections'
                prefix: 'App\Infrastructure'

```

and structure the mapping like:
```
orm/
  domain/
    User.ValueObject.Auth.Credentials.orm.yml
  projections/
    User.Query.UserView.orm.yml
```

Reply:
Ok, I will do that.